src/MineCase.Server.Interfaces/Game/OpPermissionLevel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MineCase.Server.Game
+{
+    /// <summary>
+    /// 1 - Ops can bypass spawn protection.
+    /// 2 - Ops can use all singleplayer cheats commands(except /publish, as it is not on servers; along with /debug) and use command blocks.Command blocks and Realms operators/owners have this level.
+    /// 3 - Ops can use most multiplayer-exclusive commands (all except level 4 commands).
+    /// 4 - Ops can use all commands including /stop, /save-all, /save-on, and /save-off.
+    /// </summary>
+    public enum OpPermissionLevel : byte
```

Comment:
It an acronym in minecraft. The command '\op' is the origin of it. I think most minecraft players know it, who join mc multi player servers.
https://minecraft-zh.gamepedia.com/index.php?search=op&title=Special:%E6%90%9C%E7%B4%A2&fulltext=%E6%90%9C%E7%B4%A2&searchToken=ejcq585adjnbf6a1gms3vzd47